layers/+lang/go/packages.el

```diff
@@ -39,6 +44,13 @@
             :modes go-mode
             :variables company-go-show-annotation t)))
 
+(defun go/init-lsp-go ()
+  (use-package lsp-go
+    :commands lsp-go-enable))
+
```

Comment:
Do you think we need a line `    :config (spacemacs//setup-lsp-jump-handler 'go-mode)))` here. 

I not familiar with elisp, but I got line from python layer.

Reply:
I am not sure about that, I did try that locally but it doesn't seem to do anything different. Someone with more experience should be able to comment on this.